celery/utils/time.py

```diff
@@ -383,3 +385,20 @@ def utcoffset(time=_time, localtime=_time.localtime):
 def adjust_timestamp(ts, offset, here=utcoffset):
     """Adjust timestamp based on provided utcoffset."""
     return ts - (offset - here()) * 3600
+
+
+def get_exponential_backoff_interval(
+    factor,
+    retries,
+    maximum,
+    full_jitter=False
```

Comment:
No, I don't have an actual use case. I included that version because, when I first started thinking about the concept of jitter and making a pull request to Celery, that was the algorithm that I thought of. But if research shows that the "full jitter" algorithm is superior, then I'm fine with dropping my naive algorithm.

However, we'll need to update the documentation, so that it no longer refers to a feature that doesn't exist!